poco/commands/repo_pull.py

```diff
@@ -0,0 +1,6 @@
+from .catalog_update import CatalogUpdate
+
+
+class RepoUpdate(CatalogUpdate):
+    sub_command = "repo"
```

Comment:
ez a "poco repo pull" vs "poco up planname" nem annyira egy irányba megy.
van oka hogy igy alakult? nekem elsőre jobbnak tűnne az hogy "poco pull repo"

Reply:
Azért így alakult, mert a katalogus repository-k a "repo" fő parancs alá tartoznak. Pl: "poco repo add" / "poco repo branch".

Ebből adódóan ez is a "repo" főparancs alá került, valamint elérhetővé tettem még "poco catalog-update" parancsként is.

Reply:
Oké, jogos, azokat el is felejtettem. Majd valamikor ezt is lehetne konszolidálni, hogy hova kerül az ige és a tárgy a parancsban. De ez független. 
A másikat láttam, jónak tűnik